src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -123,8 +147,39 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
         return Promise.reject(new OrderFinalizationNotRequiredError());
     }
 
+     async verifyCard(methodId: string , amount: number,  payment: any): Promise<GooglePayVerifyPayload>  {
```

Comment:
Is this supposed to be a private method?

Reply:
Also can we not use `any` as a type please.

Reply:
resolved